src/server/profiles/account/account.controller.js

```diff
@@ -0,0 +1,130 @@
+/* eslint no-unused-vars: ["error", { "argsIgnorePattern": "app" }] */
+const { resolveFromVersion } = require('../../utils/resolve.utils');
+const responseUtils = require('../../utils/response.utils');
+const errors = require('../../utils/error.utils');
+
+module.exports.getAccount = function getAccount ({ profile, logger, config, app }) {
+	let { serviceModule: service } = profile;
+
+	return (req, res, next) => {
+		let { version } = req.sanitized_args;
+		// Get a version specific resource
+		let Account = require(resolveFromVersion(version, 'uscore/Account'));
```

Comment:
So I don't think this will resolve properly, we don't have an Account resource in USCore. You might need to change this to require account from the base resource folder. e.g.

```javascript
let Account = require(resolveFromVersion(version, 'base/Account'));
```

You will probably need to make this change to every profile that is not in uscore.

Reply:
Ok